packages/json-rpc/ws-provider/src/node.ts

```diff
@@ -1,12 +1,23 @@
-// @ts-ignore
 import { WebSocket } from "ws"
-import { getInternalWsProvider } from "./ws-provider"
+import { getInternalWsProvider, type GetWsProviderInput } from "./ws-provider"
 export type { JsonRpcProvider } from "@polkadot-api/json-rpc-provider"
 
+export type {
```

Comment:
barrel type export wont work here since, rollup will epxort the internal method. Don't know why.